conans/client/cmd/export.py

```diff
@@ -104,13 +105,10 @@ def _capture_export_scm_data(conanfile, src_path, destination_folder, output, pa
     if os.path.exists(scm_src_file):
         os.unlink(scm_src_file)
 
-    try:
-        scm_data = SCMData(conanfile)
-    except ConanException:
+    scm_data = get_scm_data(conanfile)
```

Comment:
I changed my mind about the `get_scm_data` @memsharded ;)

Reply:
Perfect :) :) :)